raiden/settings.py

```diff
@@ -46,4 +46,6 @@
 ETHERSCAN_API = 'https://{network}.etherscan.io/api?module=proxy&action={action}'
 
 RED_EYES_CONTRACT_VERSION = '0.4.0'
-DEVELOPMENT_CONTRACT_VERSION = '0.3._'
+DEVELOPMENT_CONTRACT_VERSION = '0.7.0'
```

Comment:
I think it has to be 
```
DEVELOPMENT_CONTRACT_VERSION = None
```
as documented in [README](https://github.com/raiden-network/raiden-contracts/blame/a23a855650a3a9d7f135925e931d9c676784a157/README.rst#L58).

Reply:
Thanks for the heads up

Reply:
I think the Travis error is caused by this.  But `raiden-contracts` should show a better error message https://github.com/raiden-network/raiden-contracts/issues/593 about what went wrong and what can be done.